test/sharness/Makefile

```diff
@@ -16,6 +16,7 @@ all: clean deps $(T) aggregate
 clean:
 	@echo "*** $@ ***"
 	-rm -rf test-results
+	-rm -rf bin/ipfs
```

Comment:
this forces a rebuild every time. did you run into trouble? is it necessary? 

putting `$(IPFS_ROOT)/**/*.go` in the deps is most likely enough. the tests are pretty slow and part of that is that ipfs may be built multiple times. I removed a couple of these-- so lets only add it if its required